src/runtime/components/ContentRendererMarkdown.vue

```diff
@@ -116,7 +117,17 @@ function renderNode (node: MarkdownNode, h: CreateElement, documentMeta: ParsedC
   }
 
   if (node.tag === 'script') {
-    return renderToText(node)
+    return renderToString(node)
+  }
+
+  // Resolve component props
+  const props = propsToData(node, documentMeta)
+
+  // Prepend base URL to absolute images
+  if (node.tag === 'img') {
+    if (props.src?.startsWith('/') && !props.src.startsWith('//')) {
+      props.src = withBase(props.src, useRuntimeConfig().app.baseURL)
```

Comment:
Nuxt API says that a env variable can be set to change the base URL

https://nuxt.com/docs/api/configuration/nuxt-config/#baseurl